### ISSUE TITLE
Export default account (cli)

### ISSUE THREAD

**Issue by zenoxygen** [iron-fish/ironfish#24]
#### Description
Bug in ironfish-cli to export the default account.

#### Config
Default ironfish-cli configuration, with Linux 5.11.11-arch1-1.

#### To Reproduce
```
ironfish accounts:export default
TypeError: json_colorizer_1.default is not a function
```

**Comment by dguenther** [comment]
Fix for this will probably be the same as #23

**Comment by dguenther** [comment]
Closing, see #23 for related comments
